src/wasm_simd/wasm_simd_utils.rs

```diff
@@ -0,0 +1,316 @@
+use core::arch::wasm32::*;
+
+//  __  __       _   _               _________  _     _ _
+// |  \/  | __ _| |_| |__           |___ /___ \| |__ (_) |_
+// | |\/| |/ _` | __| '_ \   _____    |_ \ __) | '_ \| | __|
+// | |  | | (_| | |_| | | | |_____|  ___) / __/| |_) | | |_
+// |_|  |_|\__,_|\__|_| |_|         |____/_____|_.__/|_|\__|
+//
+
+/// Utility functions to rotate complex pointers by 90 degrees
```

Comment:
Change wording in docstring to "complex values" or "complex numbers"? 
"pointers" feels a bit odd here.

Reply:
Fixed with b1d316f